crds/nfsstorageclass.yaml

```diff
@@ -166,6 +171,16 @@ spec:
                                 type: array
                                 items:
                                   type: string
+                secureErase:
```

Comment:
Maybe give it more specific name, like `secureEraseRemainingFiles`? I'm a bit worried about users not reading documentation.

Reply:
`remainingFilesCleanupMethod` maybe even better. This way it will not interfere with Secure Erase term.